src/test/java/com/climatetree/places/dao/NameRepositoryTest.java

```diff
@@ -46,12 +46,13 @@ public void saveAndFindByIdTest() {
 		assertEquals(name.getName(), n1.getName());
 	}
 
-	@Test
-	public void getPlacesByNameTest() {
-
-		List<Name> names = new ArrayList<>();
-		repository.getPlacesByName("MAN").forEach(names::add);
+//	@Test
```

Comment:
We cannot test the repo layer here because in mem h2 database doesnt support post-gis